pex/interpreter_constraints.py

```diff
@@ -190,7 +190,7 @@ def iter_compatible_versions(
     PythonVersion(Lifecycle.EOL, 3, 5, 10),
     PythonVersion(Lifecycle.EOL, 3, 6, 15),
     PythonVersion(Lifecycle.STABLE, 3, 7, 12),
-    PythonVersion(Lifecycle.STABLE, 3, 8, 11),
+    PythonVersion(Lifecycle.STABLE, 3, 8, 12),
```

Comment:
I updated this as I contemplated making no ICs map to all interpreter versions supported by Pex. I did not make that switch, but left the update.